docs/resolvers/universalresolver.mdx

```diff
@@ -0,0 +1,58 @@
+import { WIP } from "@/components/wip/WIP";
+import { interfaceDetails } from "#/data/interfaces";
+import { universalresolver_methods } from "#/data/universalresolver";
+import { h2, h3, h4 } from '@/components/mdx/heading/h2';
+
+{/* * @type {import('@/lib/mdxPageProps').MdxMetaProps} */}
+export const meta = {
+    description: '',
+    emoji: '👉',
+    contributors: [
+        'luc.eth',
+        'serenae.eth'
+    ]
+};
+
+# Universal Resolver
+
+## Overview
+
+The Universal Resolver is a contract that handles the work of resolving a name entirely onchain, making it possible to make a single smart contract call to resolve an ENS name.
```

Comment:
Probably worth mentioning that the UR is used under the hood in viem and ensjs, and devs don't usually have to interact with it directly. This doc is for advanced use.

Reply:
Done!